app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -40,7 +40,7 @@ CONFIG_PCM_CONVERTER_FORMAT_FLOAT=y
 # SOF / audio modules / mocks
 # This mock is part of official sof-bin releases because the CI that
 # tests it can't use extra CONFIGs. See #9410, #8722 and #9386
-CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y
+CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=m
```

Comment:
just to make sure this is intended - we currently don't package loadable modules on MTL, so this will exclude this module from the binary distribution

Reply:
This is fine as this module only used for Chrome configs.